functions/index.js

```diff
@@ -1,23 +1,60 @@
-// The Cloud Functions for Firebase SDK to create Cloud Functions and setup triggers.
 const functions = require('firebase-functions');
-
-// The Firebase Admin SDK to access the Firebase Realtime Database.
 const admin = require('firebase-admin');
-admin.initializeApp();
+admin.initializeApp(functions.config().firebase);
+const express = require('express');
+const cors = require('cors')({origin: true});
+const app = express();
 
-// Create and Deploy Your First Cloud Functions
-// https://firebase.google.com/docs/functions/write-firebase-functions
-exports.helloWorld = functions.https.onRequest((request, response) => {
- response.send("Hello from Firebase!");
-});
+// TODO: Remember to set token using >> firebase functions:config:set stripe.token="SECRET_STRIPE_TOKEN_HERE"
+const stripe = require('stripe')(functions.config().stripe.token);
+
+function charge(req, res) {
```

Comment:
I would take this and split it out in a services directory:

```
.
├── app.js
└── services
    └── charges.js 
```

And then try something like this for code organization:
```js
app.post('/charges', async (req, res) => {
  const data = await ChargesService.create({ ... data });

  res.send(200)
})

class ChargesService {
  async create(data) {
    if (!data) {
      throw new Error('blah')
    }

    return stripe.charges.create({});
  }
}
```